code/modules/client/client procs.dm

```diff
@@ -371,6 +371,8 @@
 	prefs.client = src					// Safety reasons here.
 	prefs.last_ip = address				//these are gonna be used for banning
 	prefs.last_id = computer_id			//these are gonna be used for banning
+	if(world.byond_version >= 511 && byond_version >= 511 && prefs.clientfps)
+		vars["fps"] = prefs.clientfps
```

Comment:
I really don't like this implementation. The usage of vars is unneeded here and can lead to bad shit down the road.
```dm
#if DM_VERSION >= 511
	if (byond_version >= 511 && prefs.clientfps)
		fps = prefs.clientfps
#endif // DM_VERSION >= 511
```